pocs/utils/images/__init__.py

```diff
@@ -103,14 +104,22 @@ def _make_pretty_from_fits(fname, **kwargs):
         ax = plt.subplot(projection=wcs)
 
         ax.coords.grid(True, color='white', ls='solid')
-        ax.set_xlabel('RA')
-        ax.set_ylabel('Dec')
+
+        ra = ax.coords[0]
```

Comment:
This is not really an `ra`, it is an `ra_axis`. Same with the dec. Better to be explicit about the name because if you had another variable (which maybe you will in future) that actually holds an RA value you would get confused by this.